.github/workflows/create-azure-self-hosted-runners.yml

```diff
@@ -37,6 +37,9 @@ env:
   DEALLOCATE_IMMEDIATELY: ${{ github.event.inputs.deallocate_immediately }}
   # This has to be a public URL that the VM can access after creation
   POST_DEPLOYMENT_SCRIPT_URL: https://raw.githubusercontent.com/${{ github.repository }}/${{ github.ref_name }}/azure-self-hosted-runners/post-deployment-script.ps1
+  # Note that you'll need "p" (arm64 processor) and ideally "l" (local temp disk). The number 8 stands for 8 CPU-cores.
+  # For a convenient overview of all arm64 VM types, see e.g. https://azureprice.net/?_cpuArchitecture=Arm64
+  AZURE_VM_TYPE: Standard_D8plds_v5
```

Comment:
Nice thinking @dennisameling !

Could you add another knob where the Azure region can be configured, defaulting to "US West 2" (which seems to be more cost-effective than "West Europe", which is used currently)?

Reply:
Will do! Fun fact: it looks like "Central India" is ~36% cheaper than US-based VMs. Shall I default to that instead?

https://azureprice.net/vm/Standard_D8plds_v5

Reply:
> Will do! Fun fact: it looks like "Central India" is ~16% cheaper than US-based VMs. Shall I default to that instead?
> 
> https://azureprice.net/vm/Standard_D8plds_v5

Good point, let's do that!